<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserListTabPage</class>
 <widget class="QWidget" name="UserListTabPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UserListTabPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_UserListTabPage_bk" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_serachUser_bk" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>556</width>
          <height>46</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>556</width>
          <height>46</height>
         </size>
        </property>
        <widget class="SearchUserLineEdit" name="lineEdit_searchUser">
         <property name="geometry">
          <rect>
           <x>7</x>
           <y>8</y>
           <width>521</width>
           <height>28</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_searchUser">
         <property name="geometry">
          <rect>
           <x>490</x>
           <y>11</y>
           <width>21</width>
           <height>21</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_userCount">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>10</y>
           <width>431</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="UserListWidget" name="listWidget_userList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>558</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>558</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>UserListWidget</class>
   <extends>QListWidget</extends>
   <header location="global">userlistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>SearchUserLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">searchuserlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_searchUser</sender>
   <signal>clicked()</signal>
   <receiver>UserListTabPage</receiver>
   <slot>searchUserBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>searchUserBtnClicked()</slot>
 </slots>
</ui>
